bpf/lib/common.h

```diff
@@ -332,6 +332,7 @@ enum {
 #define DROP_UNKNOWN_SENDER	-172
 #define DROP_NAT_NOT_NEEDED	-173 /* Mapped as drop code, though drop not necessary. */
 #define DROP_IS_CLUSTER_IP	-174
+#define DROP_FRAG_NOT_FOUND	-175
```

Comment:
I think this needs the corresponding formatting side in `pkg/monitor/datapath_drop.go`.

Reply:
I thought more work was needed to add new metrics and that I would do a follow-up PR, but adding return code and reason are apparently enough, so sure I'll add it!